code/modules/shuttle/ftl.dm

```diff
@@ -38,9 +39,14 @@
 	var/selecting_planet = 0
 	var/do_send
 	var/icon_view_counter = 0
+	var/secondary = FALSE //For secondary Battle Bridge computers
+	var/general_quarters = FALSE //Secondary computers only work during General Quarters
+
 
 /obj/machinery/computer/ftl_navigation/New()
 	..()
+	if(secondary)
+		desc = "This is a secondary Navigation computer. It will only work during General Quarters."
```

Comment:
I like how you changed the secondary desc for the weapons computer but not the navigation computer.